integration-tests/test_client_options.py

```diff
@@ -17,3 +22,53 @@ def test_set_ansible_host_info(insights_client):
     ret = insights_client.run("--ansible-host=foo.example.com", check=False)
     assert "Could not update Ansible hostname" not in ret.stdout
     assert ret.returncode == 0
+
+
+def test_group(insights_client, tmp_path):
+    """Test --group option
+    This test generates an archive in offline mode and verifies if the group
+    specified in --group was created and packed
+    """
+
+    group_name = f"RHIQE.{uuid.uuid4()}.testing-group"
+    archive_tmp_name = f"test_archive_{uuid.uuid4()}.tar.gz"
```

Comment:
since the archive is in a temporary location created new every time the test is run, there is no need to use an unique filename for the archive; simply use `archive.tar.gz` directly

Reply:
ok both are fine so I will change it to archive.tar.gz